test/test_ui.py

```diff
@@ -737,7 +737,7 @@ def test_replacements_parsed(self):
 
         ui._raw_main(['test'])
         replacements = self.test_cmd.lib.replacements
-        self.assertEqual(replacements, [(re.compile(r'[xy]'), b'z')])
+        self.assertEqual(replacements, [(re.compile(u'[xy]'), b'z')])
```

Comment:
`r"[xy]"` is equivalent to `"[xy]"`, so this needs to be changed as well. Unfortunately there is no `ru""`-literal, but it doesn't matter.